data-reconciliation-app/demo/scripts/03-code_change_demo.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
you need to change the attributes of this file to have execute permission `chmod +x <filename>` and then commit to git

Reply:
I did this earlier but now I have done chmod a+x